.github/workflows/release-drafter.yml

```
@@ -1,4 +1,30 @@
+on:
+  push:
+    tags:
+      - 'v*'
+  pull_request:
+    types: [closed]
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)